test/post_processors/test_collector.py

```diff
@@ -0,0 +1,180 @@
+import pytest
+from curvature.collector import WayCollector
+
+class MockOSMParser(object):
+
+    nodes_callback = None
+    ways_callback = None
+    relations_callback = None
+    coords_callback = None
+
+    nodes = []
+    ways = []
+    relations = []
+    coords = []
+
+    def __init__(self, concurrency=None, nodes_callback=None, ways_callback=None,
```

Comment:
If we plan to reuse this mock for other tests it should be moved to it's own file in `lib` or `test` with perhaps even tests of it's own if it grows in complexity